lib/Gedmo/Tree/Entity/Repository/NestedTreeRepository.php

```diff
@@ -193,6 +193,114 @@ public function getPath($node)
         return $this->getPathQuery($node)->getResult();
     }
 
+    /**
+     * @see getAncestorsQueryBuilder
+     */
+    public function ancestorsQueryBuilder($node = null, $direct = false, $sortByField = null, $direction = 'ASC', $includeNode = false)
+    {
+        $meta = $this->getClassMetadata();
+        $config = $this->listener->getConfiguration($this->_em, $meta->name);
+
+        $qb = $this->getQueryBuilder();
+        $qb->select('node')
+            ->from($config['useObjectClass'], 'node')
+        ;
+        if ($node !== null) {
```

Comment:
Please, use Yoda condition.

Reply:
Because code below it'a copy-paste from getChildren. Where author of this Yoda-condition?